src/databases/functions/users/get-user-by-user-id.js

```diff
@@ -1,12 +1,14 @@
-const { User } = require("../../models")
+const {User} = require('../../models');
 
 module.exports = async (userId) => {
-    if(userId == null) return;
-    let user = await User.findOne({
-        where: {
-            user_id: userId
-        }
-    })
-    if(user == null) return []
-    return user
-}
+  if (userId == null) return;
+  const user = await User.findOne({
+    where: {
+      user_id: userId
+    },
+    raw: true
+  });
+  if (user == null) return;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Previously, when no user was found, an empty array was returned. Now, `null` is returned. This could potentially break any code that expects an array as the return value. Ensure that all calling code can handle `null`.
```diff
-  if(user == null) return []
+  if(user == null) return null;
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->